### ISSUE TITLE
Not able to learn with sinusoidal embeddings.

### ISSUE THREAD

**Issue by sarthakgarg** [facebookresearch/XLM#31]
Hi,
  I ran the MLM pretraining for en-fr using the default arguments.
I noticed that while I was able to learn using the learnt embeddings, using sinusoidal embeddings completely fails to learn and the validation accuracy stays around 5%.

Did you face similar issues while using sin embeddings too?

Thanks!

**Comment by glample** [comment]
Hi,

I have not tried sinusoidal for a very long time. I observed that it was better to learn the embeddings, so I always learn them now. But I'm surprised that this makes such a difference. So I would recommend just ignoring the sinusoidal embeddings.
